CorePayments/src/main/java/com/paypal/android/corepayments/apis/eligibility/SupportedCountryCurrencyType.kt

```diff
@@ -0,0 +1,5 @@
+package com.paypal.android.corepayments.apis.eligibility
+
+internal enum class SupportedCountryCurrencyType {
```

Comment:
Random thought: Should we make this `enum` merchant facing to prevent unsupported currency codes from being used?

Reply:
This might be a good product question. How do we want to handle adding additional currencies? Are we okay requiring an SDK update for additional currency support, or can we enable it via a serverside change?

Reply:
Yeah I could see strengths in both String and Enum-based approaches. We may be able to let the GraphQL call fail when an unsupported currency string is provided. On the other hand, an enum may be helpful to let merchants discover a list of supported currencies through auto-completion instead of having to look them up on Google, etc.

Reply:
Went ahead and removed `SupportedCountryCurrencyType` here: [1cbfa70](https://github.com/paypal/paypal-android/pull/268/commits/1cbfa7018c4bb85767f29a77028016024271aaa8)